### PR TITLE
Io500 scc 19

### PR DESCRIPTION
**Pull request by gmarkomanolis**
Merge the last changes to the master branch. 
io500.sh needs to be changed to regular if we decide to accept the merging.

We use the latest IOR and if it's one node used, we add -Y for mdtest easy/hard write.

### CODE REVIEW

**Comment by johnbent** [comment]
Hey George,

In your commit message, you say, "io500.sh needs to be changed to regular
if we decide to accept the merging."  So, perhaps you can add that now so
that everything is done in one pull request?

Julian/Jay, do you two agree that the approach George and I are on is a
good approach?

Thanks,

John

**Comment by JulianKunkel** [comment]
That is basically the approach we discussed at the phone call, so I'm good with it.
Like said, we could keep the OLD ior hash to use and bump it forward in the IOR repository!

**Comment by gflofst** [comment]
I agree with this approach too

Best,

Jay

On Oct 16, 2019, at 11:32 AM, Julian Kunkel <notifications@github.com> wrote:

﻿

That is basically the approach we discussed at the phone call, so I'm good with it.
Like said, we could keep the OLD ior hash to use and bump it forward in the IOR repository!

File: io500.sh [hunk 1]
```diff
@@ -59,8 +59,8 @@ function setup_paths {
   io500_ior_cmd=$PWD/bin/ior
   io500_mdtest_cmd=$PWD/bin/mdtest
   io500_mdreal_cmd=$PWD/bin/md-real-io
-  io500_mpirun="mpirun"
-  io500_mpiargs="-np 2"
+  io500_mpirun="jsrun"
+  io500_mpiargs="-n 6 -r 3 -a 1 -c 1"
```
Comment (johnbent) [inline io500.sh]: Are these lines correct?  I don't think these are appropriate for defaults.  Can you change these back please?

**Review by johnbent — CHANGES_REQUESTED** [review]

File: utilities/prepare.sh [hunk 2]
```diff
@@ -7,7 +7,7 @@ echo It will also attempt to build the benchmarks
 echo It will output OK at the end if builds succeed
 echo
 
-IOR_HASH=io500-sc19
+IOR_HASH=a69a5916cf0a057
```
Comment (JulianKunkel) [inline utilities/prepare.sh]: Please revert this. Let me bump the hash at the IOR repository to a69a5916cf0a057 As we figured there is an issue that affects everyone.

**Comment by johnbent** [comment]
Is it easy to create a new logical tag like io500-sc19v2 instead of
a69a5916cf0a057?

File: utilities/prepare.sh (hunk 2 above)
Comment (gmarkomanolis) [reply utilities/prepare.sh]: Done

**Review by JulianKunkel — APPROVED** [review]

**Review by gflofst — APPROVED** [review]

**Pull request merged**